src/Api/Tools/Models/Response/MemberAccessReportModel.cs

```diff
@@ -26,4 +43,103 @@ public class MemberAccessReportModel
     public int TotalItemCount { get; set; }
     public IEnumerable<MemberAccessReportAccessDetails> AccessDetails { get; set; }
 
+    /// <summary>
+    /// Generates a report for all members of an organization. Containing summary information
+    /// such as item, collection, and group counts. As well as detailed information on the
+    /// user and group collections along with their permissions
+    /// </summary>
+    /// <param name="orgId">Id of the organization to generate the report</param>
+    /// <returns>List of MemberAccessReportResponseModel</returns>
+    public static IEnumerable<MemberAccessReportResponseModel> CreateReport(
+        IEnumerable<OrganizationUserUserDetails> orgUsers,
```

Comment:
`organizationUsersTwoFactorEnabled` contains the same `OrganizationUserUserDetails` objects as `orgUsers`. I think you can remove the `orgUsers` parameter if you wanted.